src/Workspaces/Core/Portable/Editing/SyntaxEditor.cs

```diff
@@ -221,17 +290,23 @@ public override SyntaxNode Apply(SyntaxNode root, SyntaxGenerator generator)
         private class ReplaceChange : Change
         {
             private readonly Func<SyntaxNode, SyntaxGenerator, SyntaxNode> _modifier;
+            private readonly Func<SyntaxNode, SyntaxNode> _applyTrackingToNewNode;
 
-            public ReplaceChange(SyntaxNode node, Func<SyntaxNode, SyntaxGenerator, SyntaxNode> modifier)
+            public ReplaceChange(
+                SyntaxNode node,
+                Func<SyntaxNode, SyntaxGenerator, SyntaxNode> modifier,
+                Func<SyntaxNode, SyntaxNode> applyTrackingToNewNode)
```

Comment:
Do we have to pass this in? This class being private, might just call `ApplyTrackingToNewNode` directly?

Reply:
Sorry, my bad.